### ISSUE TITLE
TimeRangePicker: Cannot click relative time ranges 

### ISSUE THREAD

**Issue by torkelo** [grafana/grafana#37727]
The click area for the relative time ranges are does not extend the whole item making clicks have no effect unless you specifically hit exactly above the text.
